cli/src/lib.rs

```diff
@@ -181,6 +181,9 @@ pub enum Command {
         /// to be able to check the transactions.
         #[clap(long)]
         detach: bool,
+        /// Run the test suites under the specified path
+        #[clap(long, multiple_occurrences = true)]
```

Comment:
@Aursen In the new version of clap you upgraded to v4 vs. v3 we used to be on, `multiple_ocurrences` is no long an option. Is this because it's a default option? Same with `multiple_values` is that also just activated by default now?

Reply:
Ok, it appears that `multiple_occurences` is now actually the default behaviour in clap. So it's safe to remove.